### PR TITLE
migration data for fortios collection

### PR DESCRIPTION
**Pull request by anshulbehl**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -751,7 +751,9 @@ files:
   $modules/network/f5/bigiq_utility_license.py:
     migrated_to: f5networks.f5_modules
   $modules/network/files/: $team_networking
-  $modules/network/fortios/: bjolivot
+  $modules/network/fortios/:
```
Comment (gundalow) [inline .github/BOTMETA.yml]: Need every module listed here

File: .github/BOTMETA.yml [hunk 2]
```diff
@@ -1373,11 +1375,13 @@ files:
       - f5
   $module_utils/network/fortios:
```
Comment (gundalow) [inline .github/BOTMETA.yml:2475]: Need every module listed here

File: .github/BOTMETA.yml [hunk 3]
```diff
@@ -1373,11 +1375,13 @@ files:
...
     maintainers: bjolivot
+    migrated_to: fortinet.fortios
   $module_utils/network/fortimanager:
     maintainers: $team_fortimanager
     labels:
       - fortimanager
       - networking
+    migrated_to: fortinet.fortios
```
Comment (gundalow) [inline .github/BOTMETA.yml:2509]: Need every module listed here

File: .github/BOTMETA.yml (hunk 2 above)
Comment (anshulbehl) [reply .github/BOTMETA.yml:2475]: Resolved

File: .github/BOTMETA.yml (hunk 3 above)
Comment (anshulbehl) [reply .github/BOTMETA.yml:2509]: Resolved

File: .github/BOTMETA.yml [hunk 4]
```diff
@@ -1626,11 +2474,39 @@ files:
       - f5
   $module_utils/network/fortios:
     maintainers: bjolivot
+  $modules/network/fortios/argspec/facts/facts.py:
```
Comment (gundalow) [inline .github/BOTMETA.yml]: These directory names don't look right

Comment (gundalow) [reply .github/BOTMETA.yml]: should it be module_utils?

Comment (anshulbehl) [reply .github/BOTMETA.yml]: yeah, needs to be fixed

**Review by gundalow — APPROVED** [review]

**Pull request merged**